tests/test_math.py

```diff
@@ -46,23 +56,23 @@ def setUp(self):
 
     def test_construct_float32_default(self):
         cplx = mm.ComplexFloat32()
-        self.assertAlmostEqual(cplx.real, 0.0)
-        self.assertAlmostEqual(cplx.imag, 0.0)
+        self.assert_allclose32(cplx.real, 0.0)
```

Comment:
Use allclose for single and double precision floating point for the relative tolerance.  `assertAlmostEqual()` is implemented in a different way.